src/app/config/credentials.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "service_account",
+  "project_id": "drivenextjs",
+  "private_key_id": "386da7e455c375f89e214b4f6f5d7e8e20386d13",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQClFU1JHYI9m4No\nFzQ+i7ZPy7rp7VWrXV/58Cmg3fFOQQMGhmhBufaF83mehBAa8RJPeY0KZ/Bb1TaT\nVG7xrFFWX6ZA17Wr0UBR2SP1Ij9PnDl7EL3B7t9dKXULq+oeXtSC/bUeOiy9RMM6\nuHLS9Gc8QoK19mYqHBlYWkNRCQs8+DPl1RWykI3NRH1YuTemWkvR1W2YvpPMtQJP\nyqwdT2YYOiD4GteXaIJkt2S7rzLxdrwHMsjy3WqQs35sROuQHn6QKqKNhXETl+Ms\n/7WkhNLRdPnQ0yoXJeJXiNJgv5E5sG0Df1Vyv8qYqUGZeM4ngkn3uezzrAYMvKY1\n7Bw2jLgdAgMBAAECggEAC5Rpsi6td4RXLJ/vZHwY4FKLty5/I1Des1xxvcIWaTU9\nZixbQlGuKvm9Rs2qh9/v90lbsnFAMTTiHztdjTrCOYM77nxNwZ0L5wyKouk8KdRX\nzNvhBP+W7++Wm/27DlZ7uioxQEr3ie7kRz7FISqPK5vaaYinOcjJogKyNPbj21V3\nI79Xn47Ua5es4RC0jIF2c+aavEanOoWQ+/a8S2Hvs4sXePTgS+334hJnBb1XBh7H\nShcEmnlPNrpgZOFjaI44K3bkCLmmjiYxIDdxvnc6UkAWPAG2flE4EO0q5jxjl76M\nI3vYr5prdAy7S1TYKciSjQxdHV8AMPqBSDepEt266QKBgQDd5JV5dzvigcUjnOfI\nnuEovzO8YDMswyO9nREbhDqUzdHel/E198q3X1v3C3gPF6jiYQ6IqGIj6J7MBBSG\ntB4YcGvIV0RmB5dEMeEs1ld7AZD1PICqjwziDhIwcH/jjE7B4+018U6m3p06vuNC\njSjLN/ov1OswCORZGUUqYeHX2QKBgQC+dUY8KZjN0PJ7OI7ZK/UGgOAvQfGAJucp\nTFhZE1uxmdkC+jY/v+jexhlMiX8+y+XbRXEVnvDshjfDkU/8NQi5v0EZ0bM1x5TL\nc9YoIw0R7021fRlPz3fId1F9S/3iSQUtlVwkpnaulJ6MmjYDhNAxY0LvHBoHqoxj\nqYkOhuDb5QKBgQCOtOVhQkr0aSuFtPLo9y2uDBJU87XV8wa4exy2irs2WozAUQ4j\n/GKWvrCh2PF1G21VcsBrKYg8ue3vHiypWYwFgcAAEAgAap9hj4bXVF7c0GwPHMNP\nbkMF27DdEG1fxiwczN8Rk0lXTtLmJkB429pBoJW6T4CW2xEpMHRoRp1+OQKBgELj\nHM90LF3by8zKiCBs95hZGmJ+XjoLb5NpZH2j9sfgp/4LsZGh7D6Qg8IFZ9K9+POQ\n564YK7RlTqA4IEcKuQ3CxTI0GyP/+5mYDNX7uVNbH4dlb8Y/rmmQveWrEnat/kuL\n8NhOPdACEcx/PAjwU82EdrVSwTGAC93UZHXdmIqhAoGAMRaHfGV79bQ4nsaluWQR\nptr8BwgcNn5tXET4K6TEUSH8KxkKvJru17mgda33MDm3G6ppsqy9OcymjW7mIG9H\nRm5i+kX33S6US5QWisnLdE1JY6a4oVV+x046e3wpYW7Up5xF4/A9K1oajA7ntp0D\nkwmp2pbgkk0c9gNhZhLClfc=\n-----END PRIVATE KEY-----\n",
+  "client_email": "drivenexttest@drivenextjs.iam.gserviceaccount.com",
+  "client_id": "112772454049227083328",
```

Comment:
This file can not go here

I think we dont need all this, we may just need some .line that we can use in the .env?

Need to research

Reply:
the documentantion I was reading from google that pass the whole file